docker/images/pinot-base/pinot-base-build/ms-openjdk.dockerfile

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+ARG JAVA_VERSION=11
+ARG JDK_IMAGE=mcr.microsoft.com/openjdk/jdk
+FROM ${JDK_IMAGE}:${JAVA_VERSION}-ubuntu AS pinot_build_env
```

Comment:
`ms-openjdk.dockerfile` and `openjdk.dockerfile` are just the same except for these two lines:

```
ARG JDK_IMAGE=openjdk
FROM ${JDK_IMAGE}:${JAVA_VERSION} AS pinot_build_env
```

```
ARG JDK_IMAGE=mcr.microsoft.com/openjdk/jdk
FROM ${JDK_IMAGE}:${JAVA_VERSION}-ubuntu AS pinot_build_env
```

The differences with `amazoncoretto.dockerfile` are not that large if we also include `fontconfig`. Shouldn't be enough to have a single dockerfile base image (same with runtime) where `FROM` is build from docker args?